src/reducers/orders/order-extra-question-list-reducer.js

```
@@ -11,7 +11,7 @@
  * limitations under the License.
  **/
 import React from "react";
-import { RawHTML } from "openstack-uicore-foundation/lib/components";
+import RawHTML from "openstack-uicore-foundation/lib/components/raw-html";
```

ESLint

[error] 'RawHTML' is defined but never used.

(unused-imports/no-unused-imports)